src/ng/directive/ngOptions.js

```diff
@@ -503,7 +503,7 @@ var ngOptionsDirective = ['$compile', '$parse', function($compile, $parse) {
             option.element.selected = false;
           });
 
-          if (value) {
+          if (value && angular.isArray(value)) {
```

Comment:
You can use `isArray()` directly from inside Angular (no need for `angular.isArray()`).